tests/render/test_render_BulletListContentBlock.py

```diff
@@ -76,15 +76,27 @@ def test_all_expectations_using_test_definitions():
         len(test_files) == 61
     ), "Something went wrong when collecting JSON Expectation test fixtures"
 
+    # Not ported over to V1 so ignore for purposes of this test
+    UNSUPPORTED_EXPECTATIONS = {
+        "expect_column_pair_cramers_phi_value_to_be_less_than",
+        "expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than",
+        "expect_column_chisquare_test_p_value_to_be_greater_than",
+        "expect_column_pair_values_A_to_be_greater_than_B",
+    }
+
     # Loop over all test_files, datasets, and tests:
     test_results = defaultdict(list)
     for filename in test_files:
         test_definitions = json.load(open(filename))
-
+        if test_definitions["expectation_type"] in UNSUPPORTED_EXPECTATIONS:
+            continue
         for dataset in test_definitions["datasets"]:
             for test in dataset["tests"]:
                 # Construct an expectation from the test.
                 if type(test["in"]) == dict:  # noqa: E721
+                    # Skip tests with invalid configurations
```

Comment:
Why do we need this new `continue` conditional? This seems like a very lenient short circuit of this test.

Reply:
Because of our new call to `ExpectationConfiguration.to_domain_obj()`, we require a valid configuration for these tests. In production, a user will always have a valid configuration due to Pydantic validation, resulting in renderers always utilizing complete objects

Reply:
So this is an artifact of bad configuration in tests?
How many are there, can we fix them? Or at least get a count? Maybe we should just test some expectations vs all. I feel with this `continue` we could skip all the expectations and not be aware that we aren't testing anything.

Reply:
I misunderstood. Having `catch_exceptions` in the configuration mean we expect this test case to fail. Thanks for explaining!